LachlanMcCaskill/COS30018_Group_I_Option_A_Topic_2_Delivery_Vehicle_Routing
# Feature/environment design

Features added:
Basic Unity Scene with a field of play
Stop Prefab with it's own material

To test:
Open up Unity and open the scene called "Lachlan" and check out the prefabs folder
